src/Transport/GrpcFallbackTransport.php

```diff
@@ -90,7 +95,8 @@ public static function build(string $apiEndpoint, array $config = [])
         ];
         list($baseUri, $port) = self::normalizeServiceAddress($apiEndpoint);
         $httpHandler = $config['httpHandler'] ?: self::buildHttpHandlerAsync();
-        $transport = new GrpcFallbackTransport("$baseUri:$port", $httpHandler);
+        $logger = isset($config['logger']) ? $config['logger'] : null;
+        $transport = new GrpcFallbackTransport("$baseUri:$port", $httpHandler, $logger);
```

Comment:
consider keeping this simple
```suggestion
        $transport = new GrpcFallbackTransport("$baseUri:$port", $httpHandler, $config['logger'] ?? null);
```